### PR TITLE
Refine telemetry service name, #83

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
## Description
Refine telemetry service name

## Steps to Test
Steps to test code change

### CODE REVIEW

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java [hunk 1]
```diff
@@ -7,7 +7,9 @@
 package com.microsoft.azure.spring.cloud.autoconfigure.cache;
 
 import com.microsoft.azure.management.redis.RedisCache;
+import com.microsoft.azure.spring.cloud.autoconfigure.common.AutoConfigureUtils;
 import com.microsoft.azure.spring.cloud.autoconfigure.context.AzureContextAutoConfiguration;
+import com.microsoft.azure.spring.cloud.autoconfigure.storage.AzureStorageAutoConfiguration;
```
Comment (warrenzhu25) [inline spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java]: Why import this?

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java [hunk 2]
```diff
@@ -26,6 +28,8 @@
 import java.io.IOException;
 import java.util.Arrays;
 
+import static com.microsoft.azure.spring.cloud.autoconfigure.common.AutoConfigureUtils.getServiceName;
```
Comment (warrenzhu25) [inline spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java]: Duplicate as first added import

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java]: no idea, IDE add this, let me check if enabled auto-import.

File: spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java (hunk 2 above)
Comment (Incarnation-p-lee) [reply spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cache/AzureRedisAutoConfiguration.java]: seems IDE did this for me, let me check.

**Comment by Incarnation-p-lee** [comment]
Use constant variable as serviceName and close this PR for duplicated.

**Pull request closed without merging**